### DISCUSSION TITLE
mtex 'could not detect file format , you may ask for help in the mtex forum)

### DISCUSSION THREAD

**Discussion by yzyqq1** [mtex-toolbox/mtex#1006] [Ask Anything] [answered]
Dear experts, I have an osc file that cannot be opened in mtex. I can't solve it by checking the forum. I ask for your help，thank you！
[1.zip](https://github.com/mtex-toolbox/mtex/files/6425482/1.zip)

**Comment by kilir** [comment]
Hi,
could it be that it is a rather old osc or very much brandnew?

Osc is quite an annoying format, have a look here : https://github.com/mtex-toolbox/mtex/issues/480 or here https://github.com/mtex-toolbox/mtex/issues/498

For the moment, the easiest might be to export your data to .ang. Hope that helps.
Cheers,
Rüdiger

**Reply by yzyqq1** [reply]
Dear expert

Thank you for your answer. I have only one file and I don't know any other information. I will export it in ang format next time. Thank you again for your suggestion.

Yours sincerely

yzy

**Comment by lfgmorales** [comment]
Hi

As discussed previously, there is a chance that your *.osc file comes from a different OIM version. On the mean time, if you have access to  OIM, export the data as *. ang (and try to use this as a rule)

Cheers

Luiz

Luiz F. G. Morales
ScopeM - ETH Zürich
Sent from iPhone

On 5 May 2021, at 09:13, yzyqq1 ***@***.***> wrote:

﻿

Dear experts, I have an osc file that cannot be opened in mtex. I can't solve it by checking the forum. I ask for your help，thank you！
1.zip<https://github.com/mtex-toolbox/mtex/files/6425482/1.zip>

—
You are receiving this because you are subscribed to this thread.
Reply to this email directly, view it on GitHub<https://github.com/mtex-toolbox/mtex/discussions/1006>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/ALE5IN6DODEAIAMA7OZGGFTTMDVZLANCNFSM44EDHTIA>.

[ { ***@***.***": "http://schema.org", ***@***.***": "EmailMessage", "potentialAction": { ***@***.***": "ViewAction", "target": "https://github.com/mtex-toolbox/mtex/discussions/1006", "url": "https://github.com/mtex-toolbox/mtex/discussions/1006", "name": "View Discussion" }, "description": "View this Discussion on GitHub", "publisher": { ***@***.***": "Organization", "name": "GitHub", "url": "https://github.com" } } ]

**Reply by yzyqq1** [reply]
Dear expert

Thank you for your answer. I will try your suggestion. Thank you again.

Yours sincerely

yzy

**Comment by ralfHielscher** [comment, marked as answer]
I have looked into your osc file. The source of trouble is that MTEX finds a symmetry group `131` for copper, which it is unable to interpret. What symmetry does your EBSD system displays?

Ralf.

**Reply by yzyqq1** [reply]
Dear expert

Thank you for your time, but I don't know anything else. I have only such a simple file.

Yours sincerely

yzy

**Reply by ralfHielscher** [reply]
Extracting the attached zip file into the folder mtex/interfaces should resolve the problem.

Ralf.

[loadEBSD_osc.zip](https://github.com/mtex-toolbox/mtex/files/6425963/loadEBSD_osc.zip)

**Reply by yzyqq1** [reply]
Dear expert

It was successfully imported. You are really a great expert. Please accept my sincerest thanks！

Yours sincerely

yzy
